media/js/src/simulations/simulation4/whatAreNonlinearRegs.jsx

```diff
@@ -2,20 +2,32 @@ import React from 'react';
 import { PromptBlock } from '../../PromptBlock';
 import PropTypes from 'prop-types';
 import { Katex } from '../../utils/katexComponent';
+import dataset from './polynomial.json';
+
 
 export const WhatAreNonLinearRegressions = ({
     setshowRegLine, setshowDatasets, showRegLine, showDatasets,
     setMysteryRegLine, mysteryRegLine}) => {
 
     const DATASET_NAMES = ['Linear','Quadratic','Cubic'];
 
-    const formulas = [
-        '\\hat{y} = \\hat{\\beta}_0 + \\hat{\\beta}_1 X + u',
-        // eslint-disable-next-line max-len
-        '\\hat{y} = \\hat{\\beta}_0 + \\hat{\\beta}_1 X + \\hat{\\beta}_2 X^2 + u',
-        // eslint-disable-next-line max-len
-        '\\hat{y} = \\hat{\\beta}_0 + \\hat{\\beta}_1 X + \\hat{\\beta}_2 X^2 + \\hat{\\beta}_3 X^3 + u',
-    ];
+    const getFormula = (key) => {
+        const data = dataset[key];
+
+        if (key === 'linear') {
```

Comment:
Why not use capitalized words for the keys? Then you wouldn't have to make the conversion on Line 157

Reply:
Sure